src/index.jsx

```diff
@@ -10,6 +10,23 @@ import { remote } from 'electron'
 import { persistStore, autoRehydrate } from 'redux-persist'
 import { AsyncNodeStorage } from 'redux-persist-node-storage'
 
+// This hack makes my blood boil. But without it the intial state is
+// used instead of using the persisted state from AsyncNodeStorage. I'd prefer
+// to use something other than setTimeout to make this work, but it works for
+// now.
+const hackToLoadEverything = store => {
```

Comment:
I'm not happy with this approach, but after a bit of research I failed to find a better one. For not I think it's fine.